coordinate/marshal_test.go

```diff
@@ -0,0 +1,185 @@
+// Unit tests for marshal.go.
+//
+// Copyright 2017 Diffeo, Inc.
+// This software is released under an MIT/X11 open source license.
+
+package coordinate_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/diffeo/go-coordinate/coordinate"
```

Comment:
Indication of Diffeo's repo; this matches the layout convention we use on internal code.